odoo/addons/base/static/src/js/res_config_settings.js

```diff
@@ -342,10 +362,23 @@ var BaseSettingController = FormController.extend({
 });
 
 var BaseSettingView = FormView.extend({
+    jsLibs: [],
```

Comment:
I think you should extend jslibs (for more modularity).

Reply:
@adr-odoo  In the main FormView, there is no jsLibs so it can't be extensible. Can we declare jsLibs in the main FormView and extend it here?

Reply:
Ok, nothing to do. Thx.